.github/workflows/reusable-build.yml

```diff
@@ -133,32 +133,15 @@ jobs:
     - name: Add MSBuild to PATH
       uses: microsoft/setup-msbuild@6fb02220983dee41ce7ae257b6f4d8f9bf5ed4ce
 
-    - name: Setup choco cache folder
-      # Set the choco cache to a local folder so that it can be cached.
+    - name: Add Visual Studio LLVM to path
       if: steps.skip_check.outputs.should_skip != 'true'
-      id: choco-cache
       run: |
-        mkdir ${{github.workspace}}\choco_cache
-        choco config set --name cacheLocation --value ${{github.workspace}}\choco_cache
+        echo "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\llvm\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
```

Comment:
I don't think we should hard code path names in the middle of job commands, use variables instead (yes i see line 160 already did previously so it's not new in this PR).

Reply:
Switch to derive clang.exe path from cl.exe path.